Sources/Mocker.swift

```diff
@@ -78,14 +97,17 @@ public struct Mocker {
             shared.ignoredRules.append(rule)
         }
     }
-    
+
     /// Checks if the passed URL should be handled by the Mocker. If the URL is registered to be ignored, it will not handle the URL.
     ///
     /// - Parameter url: The URL to check for.
     /// - Returns: `true` if it should be mocked, `false` if the URL is registered as ignored.
     public static func shouldHandle(_ url: URL) -> Bool {
         shared.queue.sync {
-            return !shared.ignoredRules.contains(where: { $0.shouldIgnore(url) })
+            switch mode {
+            case .optout: return !shared.ignoredRules.contains(where: { $0.shouldIgnore(url) })
+            case .optin: return shared.mocks.contains(where: { $0.url == url })
```

Comment:
<!--
  0 failure: 
  1 warning:  Cases inside a sw...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Mocker.swift;
  Line: 109;
-->

- :warning: Cases inside a switch should always be on a newline (`switch_case_on_newline`)